docs-source/content/userguide/managing-domains/configoverrides/_index.md

```diff
@@ -338,6 +338,11 @@ spec:
 
 Incorrectly formatted override files may be accepted without warnings or errors and will not prevent WebLogic pods from booting. So, it is important to make sure that the template files are correct in a QA environment, otherwise your WebLogic Servers may start even though critically required overrides are failing to take effect.
```

Comment:
Change 'will not prevent' to 'may not prevent'.   (Due to the new fail-boot-on-error behavior)

Reply:
Done